### PR TITLE
fix: MD027/no-multiple-space-blockquote

### PR DESCRIPTION
**Pull request by nschonni**
Multiple spaces after blockquote symbol
Find: `\]\n( *)>  `
Replace: `]\n$1> `
Filter: *.md

Some manual cleanup afterwards

### CODE REVIEW

**Comment by PRMerger12** [comment]
@nschonni : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Comment by ktoliver** [comment]
@gewarren - Would you review the proposed changes? Thank you.

**Comment by gewarren** [comment]
@JasonCard We've had a number of far-reaching whitespace improvement PRs (like this one) coming through our repo. Does this pose any issues for localization?

**Comment by JasonCard** [comment]
@gewarren Thank you for checking. Removing white space will not affect loc, so no worries. The update of removed white spaces before strings will be reflected automatically and translations will be recycled as is.

File: docs/get-started/csharp/tutorial-uwp.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ First, create a Universal Windows Platform project. The project type comes with
 
    > [!NOTE]
    > If this is the first time you have used Visual Studio to create a UWP app, a **Settings** dialog box might appear. Choose **Developer mode**, and then choose **Yes**.<br><br>
-   ![Enable the Developer Mode in the UWP Settings dialog box](media/enable-developer-mode.png)<br><br>Visual Studio installs an additional Developer Mode package for you. When the package installation is complete, close the **Settings** dialog box.
+   > ![Enable the Developer Mode in the UWP Settings dialog box](media/enable-developer-mode.png)<br><br>Visual Studio installs an additional Developer Mode package for you. When the package installation is complete, close the **Settings** dialog box.
```
Comment (gewarren) [inline docs/get-started/csharp/tutorial-uwp.md]: We should check that this renders as expected on staging - https://docs.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-uwp?view=vs-2019#create-a-project

File: docs/javascript/npm-package-management.md [hunk 2]
```diff
@@ -75,8 +75,8 @@ In the window, you can use commands such as the following to install a package:
  > [!Tip]
  > By default, npm will execute in your project's home directory. If you have multiple projects
  > in your solution specify the name or the path of the project in brackets.
- `.npm [MyProjectNameOrPath] install azure@4.2.3`
+ > `.npm [MyProjectNameOrPath] install azure@4.2.3`
```
Comment (gewarren) [inline docs/javascript/npm-package-management.md]: Check staging https://docs.microsoft.com/en-us/visualstudio/javascript/npm-package-management?view=vs-2019#interactive.

**Pull request merged**